### PR TITLE
Explicitly set st2client base_url in st2_deploy.sh

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

**Comment by DoriftoShoes** [comment]
+1 this is definitely cleaner than my /etc/hosts hack.

**Comment by lakshmi-kannan** [comment]
s/0.0.0.0/127.0.0.1

**Comment by m4dcoder** [comment]
Ok. Fixed to 127.0.0.1.

**Comment by lakshmi-kannan** [comment]
LGTM

**Comment by manasdk** [comment]
+1

File: tools/st2_deploy.sh [hunk 1]
```diff
@@ -515,6 +515,13 @@ install_st2client() {
   fi
   popd
 
+  # Write ST2_BASE_URL to env
+  if [[ "$TYPE" == "rpms" ]]; then
```
Comment (Kami) [inline tools/st2_deploy.sh]: Hm, a bit confused about this part - do we still need it?

CLI config should also work on Fedora just fine (but yeah, unlike global bashrc we are using here, CLI config is specific to a system user).

**Pull request merged**